resources/tentative/todomvc-react-18/src/logic/todo-loader.ts

```diff
@@ -0,0 +1,17 @@
+import type { Params } from "react-router-dom";
+import { getTodos } from "./todo-model";
+
+export function loader({ params }: { params: Params<"filter"> }) {
```

Comment:
super duper nit: the name `loader` threw me off... would it be possible to rename it to something like:
`getTodos` or `getFilteredTodos` ?